pkg/services/queryhistory/query_history_service.go

```diff
@@ -0,0 +1,62 @@
+package queryhistory
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/grafana/grafana/pkg/components/simplejson"
+	"github.com/grafana/grafana/pkg/models"
+	"github.com/grafana/grafana/pkg/services/sqlstore"
+	"github.com/grafana/grafana/pkg/setting"
+	"github.com/grafana/grafana/pkg/util"
+)
+
+func ProvideService(cfg *setting.Cfg, sqlStore *sqlstore.SQLStore) *QueryHistoryService {
+	return &QueryHistoryService{
```

Comment:
According to [this](https://github.com/grafana/grafana/blob/main/contribute/architecture/backend/services.md#services) example it would be better calling `isDisabled()` inside `ProvideService()`.